test/powershell/Modules/Microsoft.PowerShell.Management/Remove-Item.Tests.ps1

```diff
@@ -138,4 +136,39 @@ Describe "Remove-Item" -Tags "CI" {
 	    Test-Path $testdirectory | Should -BeFalse
 	}
     }
+
+    Context "Alternate Data Streams should be supported on Windows" {
+      BeforeAll {
+        if (!$IsWindows) {
+          return
+        }
+        $fileName = "ADStest.txt"
+        $streamName = "teststream"
+        $dirName = "ADStestdir"
+        $fileContent =" This is file content."
+        $streamContent = "datastream content here"
+        $streamfile = Join-Path -Path $testpath -ChildPath $fileName
+        $streamdir = Join-Path -Path $testpath -ChildPath $dirName
+
+        $null = New-Item -Path $streamfile -ItemType "File" -force
+        Add-Content -Path $streamfile -Value $fileContent
+        Add-Content -Path $streamfile -Stream $streamName -Value $streamContent
+        $null = New-Item -Path $streamdir -ItemType "Directory" -Force
+        Add-Content -Path $streamdir -Stream $streamName -Value $streamContent
+      }
+      It "Should completely remove a datastream from a file" -Skip:(!$IsWindows) {
+        # First, we verify that the stream exists.
```

Comment:
Please remove obvious comments and use blank lines to separate logical blocks.

Reply:
Fixing.